spec/video_tag_spec.rb

```diff
@@ -64,7 +64,7 @@ def controller
           :crop         => :scale,
           :height       => "200",
           :width        => "400",
-          :video_codec  => { :codec => 'h264' },
+          :video_codec  => { :codec => 'h264', profile: 'basic', level: '3.1', b_frames: false },
```

Comment:
Change in this line does not affect assertion, please fix.

Reply:
@const-cloudinary Done.